docs/release/release-tasks.md

```diff
@@ -383,24 +383,26 @@ Information can be distributed via:
 
 * `sig-cluster-lifecycle` mailing list
   * Note: The person sending out the email should ensure that they are first part of the mailing list. If the email is sent out is not received by the community, reach out to the maintainers to unblock and approve the email.
-* Slack
+* #cluster-api Slack channel
 * Office hours
+* Release Team meetings
 * Cluster API book
-* ...
+* Conference talks
+* Github Issue (when communicating beta release to providers)
```

Comment:
Can we add a link here to clarify?

Reply:
Will add a ref to https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/release/release-tasks.md#communicate-beta-to-providers, if that is what you meant?

Reply:
Thanks, done